etc/initdev/Vagrantfile

```diff
@@ -4,7 +4,7 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "ubuntu/vivid64" # 15.04
+  config.vm.box = "bento/ubuntu-16.04"
```

Comment:
It seems to be the preferred place maintained by open sources. 

Refer to the last comment from @sethvargo in https://github.com/mitchellh/vagrant/issues/7155